Documentation/design-docs/framework-version-resolution.md

```diff
@@ -422,15 +422,15 @@ The new behavior will be to treat all settings of `rollForwardOnNoCandidateFx` t
 ### Pre-release will roll forward
 The existing behavior is that pre-release only rolls forward to the closest higher pre-release of the same `major.minor.patch`. This also means that if there's an exact match available, pre-release doesn't roll forward. Pre-release also never rolls forward to release.
 
-With the proposed behavior pre-release will be treated the same way as release: roll forward on patches, minor or major based on the settings. The only difference is that it will always consider both release and pre-release versions (unlike release which prefers to roll forward to release).  
+With the proposed behavior pre-release will be allowed to roll forward to release. If the algorithm looking for closest match finds a pre-release version it will not apply automatic roll to latest patch.
```

Comment:
```suggestion
With the proposed behavior pre-release will be allowed to roll forward to release. If the algorithm looking for closest match finds a pre-release version, it will not apply automatic roll to latest patch.
```